### PR TITLE
Problem 2 solution add

### PR DESCRIPTION
**Pull request by veliduran**

### CODE REVIEW

File: dotnet/src/Problem2.cs [hunk 1]
```diff
@@ -0,0 +1,30 @@
+using System;
+
+namespace ProjectEuler
+{
+    //By considering the terms in the Fibonacci sequence whose values do not exceed four million, find the sum of the even-valued terms.
```
Comment (UmutComlekci) [inline dotnet/src/Problem2.cs]: You should write question text like;
/*
 * [<QuestionText>]
 */

File: dotnet/src/Problem2.cs [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+                if(second %2 == 0)
+                {
+                sumOfEvenNumbs+=second;
+                }
+                GetNextFibonacciNumbers(ref first,ref second);
+            }while(second<4000000);
+            return sumOfEvenNumbs;
+        }
+       private static void GetNextFibonacciNumbers(ref int first,ref int second)
+        {
```
Comment (UmutComlekci) [inline dotnet/src/Problem2.cs]: You can improve this method. I suggest you should look at the "Recursive Method"

File: dotnet/src/Problem2.cs [hunk 3]
```diff
@@ -0,0 +1,30 @@
+using System;
+
+namespace ProjectEuler
+{
+    //By considering the terms in the Fibonacci sequence whose values do not exceed four million, find the sum of the even-valued terms.
+    public static class Problem2
+    {
+        public static int GetFibonacciEvenNumbersSum()
+        {
```
Comment (UmutComlekci) [inline dotnet/src/Problem2.cs]: There is a lot of unnecessary variables in your code. No need for this variables.

File: dotnet/src/Problem2.cs (hunk 2 above)
Comment (veliduran) [reply dotnet/src/Problem2.cs]: Thanks for your advice, I have improved method with recursive type.

File: dotnet/src/Problem2.cs [hunk 4]
```diff
@@ -2,29 +2,26 @@
...
 namespace ProjectEuler
 {
-    //By considering the terms in the Fibonacci sequence whose values do not exceed four million, find the sum of the even-valued terms.
+    /*
+    *By considering the terms in the Fibonacci sequence whose values do not exceed four million, find the sum of the even-valued terms.
+    */ 
     public static class Problem2
     {
-        public static int GetFibonacciEvenNumbersSum()
+        public static int GetFibonacciEvenNumbersSum(int a, int b,int sum)
```
Comment (UmutComlekci) [inline dotnet/src/Problem2.cs]: Format your curly braces and all other codes. Change the variables name with more meaningfully.

**Pull request merged**